recipes-wpe/wpeframework/wpeframework-plugin-launcher_git.bb

```diff
@@ -4,6 +4,6 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
 
 require include/wpeframework-plugins.inc
 
-SRC_URI = "git://git@github.com/WebPlatformForEmbedded/WPEPluginLauncher.git"
+SRC_URI = "git://github.com/WebPlatformForEmbedded/WPEPluginLauncher.git"
```

Comment:
"protocol=git" maybe required here, kindly check and confirm once. Otherwise LTGM.